docs/api/assets.md

```diff
@@ -0,0 +1,422 @@
+| hreflang    | `string`  | &check; | &check; | `undefined`  | Correlates to the same attribute on a HTML `<link>` element |
+| title       | `string`  | &check; | &check; | `undefined`  | Correlates to the same attribute on a HTML `<link>` element |
+| media       | `string`  | &check; | &check; | `undefined`  | Correlates to the same attribute on a HTML `<link>` element |
+| type        | `string`  | &check; | &check; | `text/css`   | Correlates to the same attribute on a HTML `<link>` element |
+| rel         | `string`  | &check; | &check; | `stylesheet` | Correlates to the same attribute on a HTML `<link>` element |
+| as          | `string`  | &check; | &check; | `undefined`  | Correlates to the same attribute on a HTML `<link>` element |
+
+## Methods
+
+An `AssetCSS` instance has the following methods:
+
+### .toJSON()
+
+Returns JSON representation of the `AssetCSS` instance.
+
+### .toHTML() {
+
+Returns a HTML `link` element as a string of the `AssetCSS` instance.
+
+## AssetJS
+
+An `AssetJS` instance holds information about a Javascript related to a podlet
+or layout.
+
+## Properties
+
+An `AssetJS` instance has the following properties:
+
+| property    | type      | getter  | setter  | default      | details                                                     |
+| ----------- | --------- | ------- | ------- | ------------ | ----------------------------------------------------------- |
+| value       | `string`  | &check; |         | `''`         | Relative or absolute URL to the CSS asset                   |
+| href        | `string`  | &check; |         | `''`         | Alias for the `value` property                              |
+| crossorigin | `string`  | &check; | &check; | `undefined`  | Correlates to the same attribute on a HTML `<link>` element |
+| disabled    | `boolean` | &check; | &check; | `false`      | Correlates to the same attribute on a HTML `<link>` element |
+| hreflang    | `string`  | &check; | &check; | `undefined`  | Correlates to the same attribute on a HTML `<link>` element |
+| title       | `string`  | &check; | &check; | `undefined`  | Correlates to the same attribute on a HTML `<link>` element |
+| media       | `string`  | &check; | &check; | `undefined`  | Correlates to the same attribute on a HTML `<link>` element |
+| type        | `string`  | &check; | &check; | `text/css`   | Correlates to the same attribute on a HTML `<link>` element |
+| rel         | `string`  | &check; | &check; | `stylesheet` | Correlates to the same attribute on a HTML `<link>` element |
+| as          | `string`  | &check; | &check; | `undefined`  | Correlates to the same attribute on a HTML `<link>` element |
+
+## Methods
+
+An `AssetJS` instance has the following methods:
+
+### .toJSON()
+
+Returns JSON representation of the `AssetJS` instance.
+
+### .toHTML() {
+
+Returns a HTML `script` element as a string of the `AssetJS` instance.
```

Comment:
Returns an HTML `script` element as a string representation of the AssetJS instance.